specs/activity-api.yaml

```diff
@@ -202,48 +173,37 @@ paths:
             type: string
         - name: commandIndex
           description: >
-            Wait until command with the specified index finishes. Must be accompanied by a valid "timeout" query parameter.
+            Wait until command with the specified index finishes. Must be accompanied by a valid "pollTimeout" query parameter.
           required: false
           in: query
           schema:
             type: number
             format: integer
             minimum: 0
-        - $ref: 'common.yaml#/parameters/timeout'
+        - $ref: 'common.yaml#/parameters/afterTimestamp'
+        - $ref: 'common.yaml#/parameters/pollTimeout'
```

Comment:
This is no longer needed due to lack of persistence

Reply:
Agreed.